package.json

```diff
@@ -4,7 +4,8 @@
   "description": "Engine used by various sites to power the games",
   "main": "index.js",
   "scripts": {
-    "test": "npm test"
+    "test": "npm test",
+    "lint": "eslint index.js lib/game_classes/ --fix"
```

Comment:
Please change the `lint` script to work on all the files, and not fix them automatically. I believe it would be `eslint ./`.